bin/start-vm

```diff
@@ -449,7 +453,7 @@ printf "  starting VM(UUID:%s) with MAC:%s in %s\n" $uuid $macfull $targetDir
 [ $pxe ]	&& printf "  pxeboot: %s.ipxe\n" $mac
 [ $pxe ]	&& printf "  pxeboot: files served from %s\n" "$pxe"
 [ $bridge ]	&& printf "  interface: %s  bridged\n" $bridge
-[ $bridge ]	|| printf "  sshport: %s  unbridged\n" $port
+[ $bridge ]	|| printf "  sshport: %s (%s) unbridged\n" $port $port_dest
```

Comment:
we should shortly describe the differences within the output, while only `support: 2223 (22) unbridged` could be a little bit confusing. Maybe:

`printf " sshport: VM tcp/%s mapped to host tcp/%s unbridged\n" $port_dest $port`
or
`printf " sshport: local %s  -> vm %s (unbridged)\n" $port $port_dest`